test/test_mps.py

```diff
@@ -1406,25 +1406,28 @@ def helper(operator):
             if operator == "<=":
                 res_mps = x_mps <= y_mps
                 res_cpu = x_cpu <= y_cpu
-            if operator == "<":
+            elif operator == "<":
```

Comment:
This seems unrelated to change in question, is it?

Reply:
Yes not much related, but I think this improvement is trivial and small enough to not cause a reading difficulty. I feel it would be more costly and unnecessary to improve this in a separate PR, wouldn't it?